cli/src/main/java/com/box/l10n/mojito/cli/command/checks/CheckerUtils.java

```diff
@@ -1,15 +1,31 @@
 package com.box.l10n.mojito.cli.command.checks;
 
 import com.ibm.icu.text.BreakIterator;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Locale;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
 
 public class CheckerUtils {
 
+  /**
+   * Pattern matching for html tags.
+   *
+   * <p>"<" matches an opening angle bracket, "[^>]+" matches one or more characters that are not a
+   * closing angle bracket and ">" matches a closing angle bracket
+   */
+  private static final Pattern HTML_TAGS_PATTERN = Pattern.compile("<[^>]+>");
```

Comment:
To note, this won't work for html attributes that are localizable but could still be spell-checked, e.g.: the "title=" attribute on links or the "alt=" attribute on images, "value=" or "placeholder=" for input tags, etc.

At this point in time, reducing false positives (<br> tags reported as misspelled) - is likely more important than not spell checking the attributes mentioned as they are much rarer - so this might be Ok for now.

@aurambaj if you have any thoughts here.

Reply:
Thanks for your feedback George. It might not be easy to properly identify when strings inside html tags should be considered for spell checking or not. Since this rarely happens, let me know if any of you think this is worth spending time on.

Reply:
@geoarion didn't you look into using a library to extract words from html text like jsoup? worth pushing more on that you think? We could also restrict to just simple tags (no attribute) which I think were the most reported. Not sure how many reports we had on ones with attributes.

Reply:
@aurambaj The default behaviour from JSoup's text() method on a parsed html Document to extract text from html is equivalent to this - though they do provide the list of elements and attributes that could be iterated on to also get the text from the attributes (or from a pre-approved list of attributes).

That said, the number of instances of misspelled text in html attributes is likely to be small / might not be worth effort to cover.